src/opaque.rs

```diff
@@ -1158,7 +712,36 @@ impl<CS: CipherSuite> TryFrom<&[u8]> for ServerLogin<CS> {
     }
 }
 
-type ServerLoginStartResult<CS> = (LoginSecondMessage<CS>, ServerLogin<CS>);
+/// Optional parameters for server login start
+pub enum ServerLoginStartParameters {
```

Comment:
nit: this could actually implement (and use) [`core::default::Default`](https://doc.rust-lang.org/core/default/trait.Default.html)